ansible_base/lib/utils/validation.py

```
@@ -17,6 +17,40 @@
 
 VALID_STRING = _('Must be a valid string')
 
+DEFAULT_NAME_FIELDS = frozenset({'name', 'username', 'hostname'})
+
+RESOURCE_NAME_RE = re.compile(r'^[\w][\w .@-]{0,511}\Z')
+
+CONTROL_CHARS = '[\x00-\x08\x0b\x0c\x0d-\x1f\x7f-\x9f\u200b-\u200c\u200e-\u200f\u2028-\u202e\ufeff\ufff9-\ufffb]'
+
+DANGEROUS_PATTERNS = re.compile(
+    CONTROL_CHARS + r'|[<＜]\s*/?(?:script|iframe|object|embed|form|base|meta|link|svg|math|template)\b'
```

Ruff (0.16.1)

[warning] String contains ambiguous `＜` (FULLWIDTH LESS-THAN SIGN). Did you mean `<` (LESS-THAN SIGN)?

(RUF001)